Client/Frontend/Fakespot/Views/HighlightsCardView.swift

```diff
@@ -0,0 +1,75 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import ComponentLibrary
+import Common
+import UIKit
+
+struct HighlightsCardViewModel {
+    let cardA11yId: String = AccessibilityIdentifiers.Shopping.HighlightsCard.card
+    let footerTitle: String
+    let footerActionTitle: String
+    let footerA11yTitleIdentifier: String = AccessibilityIdentifiers.Shopping.HighlightsCard.footerTitle
+    let footerA11yActionIdentifier: String = AccessibilityIdentifiers.Shopping.HighlightsCard.footerAction
+
+    var footerModel: ActionFooterViewModel {
+        return ActionFooterViewModel(title: footerTitle,
+                                     actionTitle: footerActionTitle,
+                                     a11yTitleIdentifier: footerA11yTitleIdentifier,
+                                     a11yActionIdentifier: footerA11yActionIdentifier)
+    }
+}
+
+class HighlightsCardView: UIView, ThemeApplicable {
+    private struct UX {
+        static let cardBottomSpace: CGFloat = 16
+        static let footerHorizontalSpace: CGFloat = 16
+    }
+
+    private lazy var cardContainer: ShadowCardView = .build()
+    private lazy var contentView: UIView = .build()
+    private lazy var footerView: ActionFooterView = .build()
+
+    private var viewModel: HighlightsCardViewModel?
+
+    // MARK: - Inits
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        setupLayout()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
```

Comment:
I know we don't use storyboards so this method is not supposed to be called, but in other places we did `setupLayout` so we don't have fatalErrors around

Reply:
In about 90-95% we use the fatal error - so I went with what is more prevalent in our code base.

Reply:
Personally I prefer fatal error to failing silently. It means if someone does something unexpected in the future our UI tests are very likely to catch it because it will crash.